tests/test_trainer.py

```diff
@@ -176,3 +189,12 @@ def test_trainer_eval_lm(self):
             tokenizer=tokenizer, file_path=PATH_SAMPLE_TEXT, block_size=tokenizer.max_len_single_sentence,
         )
         self.assertEqual(len(dataset), 31)
+
+    def test_trainer_iterable_dataset(self):
```

Comment:
Please use the appropriate "tiny" model from  https://huggingface.co/models?search=tiny. The test should run in < 1s. You can time it by passing `--durations 0` to pytest.